### PR TITLE
remove 'app' and 'namespace' labels from remotewrite nginx graphs

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**
This PR removes the `app` and `namespace` labels from the prometheus-remotewrite's nginx graphs as they're preventing those graphs from working properly

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Can you post a screenshot ?

**Review by QuentinBisson — APPROVED** [review]

**Comment by QuantumEnigmaa** [comment]
> Can you post a screenshot ?

Sure, let me deploy this branch on `grizzly ` to show the diff :)

**Comment by QuantumEnigmaa** [comment]
With current dashboard release : 
image

With this branch : 
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - The private Zot dashboard is updated because of a namespace change, and some minor fixes are applied.
+- Remove `app` and `namespace` labels from the prometheus - remotewrite's nginx graphs.
```
Comment (QuentinBisson) [inline CHANGELOG.md:23]: That's a dupe right?

Comment (QuantumEnigmaa) [reply CHANGELOG.md]: Yeah sorry, just removed it :+1:

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**